### ISSUE TITLE
cloud backup (an automatic way to share rules of more pc)

### ISSUE THREAD

**Issue by pol-hallen** [RealRaven2000/quickFilters#111]
Hello,
can you add an easy way (google drive cloud?) to automatic share rules of addon to many pc?

**Comment by avmaksimov** [comment]
I think better to add ability to auto export rules after changes. It's easier and can be used in any cloud storage.

**Comment by RealRaven2000** [comment]
This stuff is difficult, but I get the request once in a while. I think we should start with adding a request of backing up _all account settings_ in a single file first.

**Comment by EnigmaSolved** [comment]
I get the impression that Sync (a la Firefox Sync) is coming to Thunderbird which makes me wonder if eventually Sync could be used to sync extension settings/configs like this?

**Comment by RealRaven2000** [comment]
> I get the impression that Sync (a la Firefox Sync) is coming to Thunderbird which makes me wonder if eventually Sync could be used to sync extension settings/configs like this?

Yes then it would be possible, I would preferably use it once the mail extension APIs support it

**Comment by RealRaven2000** [comment]
> I get the impression that Sync (a la Firefox Sync) is coming to Thunderbird which makes me wonder if eventually Sync could be used to sync extension settings/configs like this?

I think Firefox Sync will eventually land in Thunderbird, maybe as early as Thunderbird 128 this summer!

**Comment by EnigmaSolved** [comment]
> I think Firefox Sync will eventually land in Thunderbird, maybe as early as Thunderbird 128 this summer!

Wow, that is awesome news!!
